### ISSUE TITLE
Misaligned Rows in DataTable with Frozen Columns 

### ISSUE THREAD

**Issue by walllzzz** [primefaces/primefaces#1155]
Hi, 
I want to get back to the issue number #644, to resume the issue, when we add "frozencolumns" property to the datatable with rows with different heights, we notice a misalignment in the rows of the datatable.

the user @cnsgithub resumed the issue in two points:
1. If any row on the scrollable part is bigger than the rows on the fixed/frozen part, they become misaliged
2. If any header row on the scrollable part is bigger than the header on the fixed part, they become misaliged. 

Well, I tried to reproduce the issue with a very basic code, the first one is a very basic datable without any frozen columns, and with row height different from column to another : 

```
<p:dataTable value="#{viewEmployeesManagedBean.employees}" var="employee">
    <p:column>
        <f:facet name="header">
            <h:outputText value="Employee ID"></h:outputText>
        </f:facet>
        <h:outputText value="THIS IS A VERY VERY VERY VERY VERY
        VERY VERY
         VERY VERY 
          VERY VERY 
           VERY VERY BIG 
        COLUMN VALUE"></h:outputText>
    </p:column>
    <p:column>
        <f:facet name="header">
            <h:outputText value="Employee First Name"></h:outputText>
        </f:facet>
        <h:outputText value="#{employee.employeeName}"></h:outputText>
    </p:column>
    <p:column>
        <f:facet name="header">
            <h:outputText value="Employee Last Name"></h:outputText>
        </f:facet>
        <h:outputText value="#{employee.employeeName}"></h:outputText>
    </p:column>
</p:dataTable>
```

this is the output, everything is perfect: 

![normal datatable](https://cloud.githubusercontent.com/assets/1905938/13075134/59eeea1c-d4a9-11e5-91fb-32a100048153.PNG)
- When adding frozen columns property to the first column and "scrollable" to true: 

```
<p:dataTable scrollable="true" frozenColumns="1" value="#{viewEmployeesManagedBean.employees}" var="employee">
    <p:column>
        <f:facet name="header">
            <h:outputText value="Employee ID"></h:outputText>
        </f:facet>
        <h:outputText value="THIS IS A VERY VERY VERY VERY VERY
        VERY VERY
         VERY VERY 
          VERY VERY 
           VERY VERY BIG 
        COLUMN VALUE"></h:outputText>
    </p:column>
    <p:column>
        <f:facet name="header">
            <h:outputText value="Employee First Name"></h:outputText>
        </f:facet>
        <h:outputText value="#{employee.employeeName}"></h:outputText>
    </p:column>
    <p:column>
        <f:facet name="header">
            <h:outputText value="Employee Last Name"></h:outputText>
        </f:facet>
        <h:outputText value="#{employee.employeeName}"></h:outputText>
    </p:column>
</p:dataTable>
```

we get the issues that I mentioned above, misaligned rows : 
![datatable with frozen column](https://cloud.githubusercontent.com/assets/1905938/13075186/b2c5acac-d4a9-11e5-9a54-ec9917477a54.PNG)

I know that there is a workaround using javascript, but I want to do a pull request to fix this.

**Comment by kukel** [comment]
1 is in reverse to!!! if one of the you should always take the biggest size being either in the frozen or in the scrollable part. Please take that into account in your 'fix'

**Comment by schlebe** [comment]
I have the same issue with simple DataTable and I can explain Why ?

On an existing DataTable, I have added the following line of HTML Primefaces code

frozenColumns="1"

When the table is displayed, the rows of the first column are not more aligned with the row of the rest of the table.

The problem happens because the presence of icon in second part of table make that the rows in right part of table are higher that the row of frozen part of table.

I have tested with Primefaces 6.1.0 and 6.1.1.

I know like wlllzzz that there exists certainly a CSS workaround, but I consider this issue like a bug.

A acceptable workaround would be to define a rowHeight attribute !

A professional solution will be to fix this bug.
